hoomd/md/methods.py

```diff
@@ -225,251 +225,79 @@ class npt(_Method):
     A :py:class:`hoomd.compute.thermo` is automatically specified and associated with *group*.
 
     Examples::
-
-        integrate.npt(group=all, kT=1.0, tau=0.5, tauP=1.0, P=2.0)
-        integrator = integrate.npt(group=all, tau=1.0, kT=0.65, tauP = 1.2, P=2.0)
+        all = filter.All()
```

Comment:
Do not use variable name `all` this is bad practice. Use `all_` or just call `filter.All()` where needed.